test/integration/rules/aria-empty-table-header/aria-empty-table-header.json

```diff
@@ -0,0 +1,6 @@
+{
+	"description": "aria-empty-table-header tests",
+	"rule": "aria-empty-table-header",
+	"violations": [["fail1"], ["fail2"], ["fail3"]],
```

Comment:
Now that the rule no longer returns violations this should be `incomplete`. Also, the selectors need the `#` in front to match the id.

Reply:
So it should be "incomplete": [["#incomplete1"], etc", cmiiw?

Reply:
Sure, updating the ids would be nice too.